src/Toolkit/A.php

```diff
@@ -47,6 +47,21 @@ public static function apply(array $array, ...$args): array
 		return $array;
 	}
 
+	/**
+	 * Counts the number of elements in an array
+	 *
+	 * @param array $array
+	 * @return int
+	 */
+	public static function count(array $array = []): int
+	{
+		if (!is_array($array)) {
+			throw new Exception('Invalid argument type: ' . gettype($array));
+		}
+
+		return count($array);
+	}
```

Comment:
Argument is `array $array = []`  and control is `!is_array($array)` 😅

Reply:
Leftover from when I haven't realized php type checking will catch the error. Will remove